Makefile

```diff
@@ -178,14 +197,14 @@ GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
```

Comment:
Let's include also a kustomize version update to [4.5.7](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.7).